dotcom-rendering/src/components/EmailSignUpSwitcher.tsx

```diff
@@ -19,12 +18,8 @@ export const EmailSignUpSwitcher = ({
 	const { renderingTarget } = useConfig();
 
 	return renderingTarget === 'Apps' ? (
-		<Island priority="feature" defer={{ until: 'idle' }}>
```

Comment:
this component doesnt need to be an island anymore as we removed the bridget check